CRM/Report/Form/Walklist/Walklist.php

```diff
@@ -227,11 +227,11 @@ public function orderBy() {
     foreach ($this->_columns as $tableName => $table) {
       if (array_key_exists('order_bys', $table)) {
         foreach ($table['order_bys'] as $fieldName => $field) {
-          $this->_orderBy[] = $field['dbAlias'];
```

Comment:
I don't want to try that in this PR as i think that is a separate issue to deal with, This is just about making sure these reports pass php7.1